data-prepper-plugins/s3-source/src/test/java/org/opensearch/dataprepper/plugins/source/SqsWorkerTest.java

```diff
@@ -245,4 +247,30 @@ void processSqsMessages_should_invoke_delete_if_input_is_not_valid_JSON_and_dele
         verify(sqsMessagesDeletedCounter).increment(1);
         verify(sqsMessagesFailedCounter).increment();
     }
+
+    @Test
+    void populateS3Reference_should_interact_with_getUrlDecodedKey() throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
+        // Using reflection to unit test a private method as part of bug fix.
```

Comment:
Minor and non-blocking: It makes sense to include a link to the Bug for future reference